gulpfile.js

```diff
@@ -440,7 +468,7 @@ gulp.task('build-langfiles', function(done) {
  *     msg/json/*.js
  */
 gulp.task('build', gulp.parallel(
-  'build-core',
+  'build-compressed',
```

Comment:
Maybe call build-core and build-generators from here?

Reply:
done